resultset.go

```diff
@@ -10,6 +10,10 @@ import (
 // not allowed.
 var ErrRawScan = fmt.Errorf("result set comes from raw query, use RawScan instead")
 
+// ErrStop is an error returned by a callback to stop the iteration in a
```

Comment:
you can remove this one from here, I added it somewhere else on the codegen branch (or I can delete mine and move this one to `store.go`)